### PR TITLE
Add openai.audio connector

### PR DESCRIPTION
**Pull request by oshadajay**
# Description

Add openai.audio connector.

Fixes: https://github.com/wso2-enterprise/choreo/issues/20370

### CODE REVIEW

File: openapi/openai.audio/Module.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+```ballerina
+public function main() returns error? {
+    audio:CreateTranscriptionRequest req = {
+        file: {fileContent: check io:fileReadBytes("sample.mp3"), fileName: "sample.mp3"},
+        model: "whisper-1"
+    };
+    audio:CreateTranscriptionResponse res = check audioClient->/audio/transcriptions.post(req);
+    io:println(res.text);
+}
+```
```
Comment (NipunaRanasinghe) [inline openapi/openai.audio/Module.md:48]: Lets indent this block to be placed under the 1st item of the numbered list

Comment (oshadajay) [reply openapi/openai.audio/Module.md]: Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/772/commits/fbea253cff693d8358f2dd78b4bd393cca40b97b.

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Comment by abeykoon** [comment]
LGTM

**Review by abeykoon — APPROVED** [review]

**Pull request merged**